.github/scripts/prepare-quarkus-next.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+git checkout -b new-quarkus-next origin/main
+
+if ! grep -q '<repositories>' pom.xml; then
+  sed -i '/<\/project>/i \
+          <repositories> \
+          </repositories>' pom.xml
+fi
+
+# Insert the <repository> element before the closing </repositories> tag
+sed -i '/<\/repositories>/i \
+        <repository> \
+        <id>sonatype-snapshots</id> \
+        <name>Sonatype Snapshots</name> \
+        <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url> \
+        <snapshots> \
+            <enabled>true</enabled> \
+            <updatePolicy>daily</updatePolicy> \
+        </snapshots> \
+        <releases> \
+            <enabled>false</enabled> \
+        </releases> \
+        </repository>' pom.xml
+
+./mvnw versions:set-property -Dproperty=quarkus.version -DnewVersion=999-SNAPSHOT
+./mvnw versions:set-property -Dproperty=quarkus.build.version -DnewVersion=999-SNAPSHOT
+git commit -am "Set quarkus version to 999-SNAPSHOT"
+
+snapshot_version_hash=$(git log origin/quarkus-next --grep="Set quarkus version to 999-SNAPSHOT" --format="%H" -n 1)
+commits_to_cherry_pick=$(git rev-list --right-only --no-merges --reverse new-quarkus-next...origin/quarkus-next | grep -vE "$snapshot_version_hash" || echo "")
+
+if [ -n "$commits_to_cherry_pick" ]; then
+  git cherry-pick "$commits_to_cherry_pick"
```

Comment:
I think this won't work with more than 1 commit. AFAIK the delimiter between commit hashes need to be spaces, not new lines (which is the output of `git rev-list`).

Reply:
It worked for me, at least while I was testing it, also this [branch](https://github.com/keycloak/keycloak/compare/main...Pepo48:keycloak:quarkus-next) is a proof that more than one commit got cherrypicked. That being said, I might've easily overlooked something, so let me, please, check it once again.

Reply:
@vmuzikar I addressed this one too (I was able to reproduce it after all).